nx-dev/nx-dev/project.json

```diff
@@ -68,10 +45,10 @@
         "production": {}
       }
     },
-    "serve": {
+    "serve-nextjs": {
       "executor": "@nrwl/next:server",
       "options": {
-        "buildTarget": "nx-dev:build",
+        "buildTarget": "nx-dev:build-nextjs",
```

Comment:
This target is missing, should be `nx-dev:build-base`, same with the prod  config.